server/src/db/database.helper.js

```diff
@@ -0,0 +1,50 @@
+const axios = require('axios');
+const databaseWriteUrl = 'https://zccore.herokuapp.com/data/write';
+const databaseReadUrl = 'https://zccore.herokuapp.com/data/read';
+
+class DatabaseConnection {
+  constructor(collection_name) {
+    this.data = {
+      plugin_id: '6132482f569dbbb7ce5b4fe5', //registered chess plug-in
+      organization_id: '612a3a914acf115e685df8e3',
+      collection_name: "chess_room",
+      bulk_write: false,
+      object_id: '',
+      filter: {},
+      payload: {}
+    }
+  }
+
+  create = async (payload) => {
```

Comment:
Refactor the methods/functions in this class to have the same signature (style) as the ones in other (controller) classes. 
e.g
```
async create(payload){
//logic
}
```